app/src/main/java/com/example/testcompras/iu/viewModel/MainViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.testcompras.iu.viewModel
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.testcompras.domain.GetDataUseCase
+
+class MainViewModel(private val getTextUseCase: GetDataUseCase) : ViewModel() {
+
+    val _dataModel = MutableLiveData<String>()
```

Comment:
this has to be private

please change and explain to me why it have to be private